app/presenters/renewal_letter_presenter.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class RenewalLetterPresenter < BaseLetterPresenter
+  # Provides the full postal address for the letter.
+  def postal_address_lines
+    lines = [contact_full_name]
+    lines << operator_name unless business_type == WasteExemptionsEngine::Registration::BUSINESS_TYPES[:sole_trader]
+    lines << address_lines(contact_address)
+    lines.flatten!.reject(&:blank?)
+  end
+
+  def exemption_description(exemption)
+    "#{exemption.code} #{exemption.summary}"
+  end
+
+  def expiry_date
+    # Currently you can only add exemptions when you register, so we can assume they expire at the same time
+    registration_exemptions.first.expires_on.to_formatted_s(:day_month_year)
+  end
+
+  def listable_exemptions
+    @_listable_exemptions ||= exemptions.first(18)
+    @_listable_exemptions
+  end
+
+  def unlisted_exemption_count
+    @_unlisted_exemption_count ||= calculate_number_of_unlisted_exemptions
+  end
+
+  def partners_list
+    return unless partnership?
```

Comment:
```suggestion
```
I don't think we need to do this at all. The `partnership?` check already happen in the view. In the `worst case` it make anyway sense for this method to return an empty array when this is not a partnership rather than `nil`. So that if you expect a method to return an array you can check if the array contains object rather than having to remember that it can also return a `nil` object, which won't respond to methods like `count` and `size`.

Reply:
❤️ We might have to work through the impact to ensure we don't break anything but I think @cintamani is right. My bad, think this was my code originally 🤦‍♂ 😳

Reply:
Resolved by https://github.com/DEFRA/waste-exemptions-back-office-ta/pull/357/commits/5f6b9a1ac8b05dd7936852a84bdeb554c9bf2e48